Backend/src/test/java/com/senla/repository/FriendshipRepositoryTest.java

```diff
@@ -0,0 +1,81 @@
+package com.senla.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.senla.model.Friendship;
+import com.senla.model.User;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+
+class FriendshipRepositoryTest extends DatabaseTest {
+
+    @Autowired private UserRepository userRepository;
+
+    @Autowired private FriendshipRepository friendshipRepository;
+
+    @Test
```

Comment:
You can put spaces between different part of the test. 
Comments can be skiped.
```java
@Test
    void findFriendship() {
        // given
        Optional<User> user1 = userRepository.findByEmail("admin@gmail.com");
        Optional<User> user2 = userRepository.findByEmail("user@gmail.com");
        Friendship friendship =
                Friendship.builder()
                        .sender(user1.get())
                        .receiver(user2.get())
                        .accepted(Boolean.TRUE)
                        .build();
        friendshipRepository.save(friendship);
        
        // when
        Optional<Friendship> foundFriendship =
                friendshipRepository.findFriendship(user1.get().getId(), user2.get().getId()); 
        // then         
        assertThat(foundFriendship.get()).isNotNull();
    }
```